hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -726,26 +722,19 @@
             <interface>10.10.1.*</interface>
         </interfaces>
         <ssl enabled="false">
-            <factory-class-name>com.hazelcast.nio.ssl.BasicSSLContextFactory</factory-class-name>
             <properties>
                 <property name="protocol">TLS</property>
 
                 <property name="mutualAuthentication">REQUIRED</property>
                 <property name="keyStore">/opt/hazelcast.keystore</property>
                 <property name="keyStorePassword">secret.97531</property>
-                <property name="keyStoreType">JKS</property>
+                <property name="keyStoreType">PKCS12</property>
                 <property name="trustStore">/opt/hazelcast.truststore</property>
                 <property name="trustStorePassword">changeit</property>
-                <property name="trustStoreType">JKS</property>
+                <property name="trustStoreType">PKCS12</property>
             </properties>
         </ssl>
         <socket-interceptor enabled="false"/>
-        <symmetric-encryption enabled="false">
-            <algorithm>PBEWithMD5AndDES</algorithm>
-            <password>...</password>
-            <salt>...</salt>
-            <iteration-count>7</iteration-count>
-        </symmetric-encryption>
```

Comment:
It is deprecated, but still, it is the full example. Looking at the other deprecated configs, like hot-restart, they are not present - is it a convention that we don't list deprecated stuff here?

Reply:
AFAIK we have no policy or convention. I want to make clear what and how to use.

The `symmetric-encryption` example is still left in the commented part, together with the deprecation note.